test/end-to-end/thirdpartypasswordless.test.js

```diff
@@ -193,6 +193,8 @@ describe("SuperTokens Third Party Passwordless", function () {
                 "ST_LOGS THIRDPARTYPASSWORDLESS PRE_API_HOOKS PASSWORDLESS_CREATE_CODE",
                 "ST_LOGS THIRDPARTYPASSWORDLESS ON_HANDLE_EVENT PASSWORDLESS_CODE_SENT",
                 "ST_LOGS THIRDPARTYPASSWORDLESS OVERRIDE SET_LOGIN_ATTEMPT_INFO",
+                "ST_LOGS THIRDPARTYPASSWORDLESS OVERRIDE GET_LOGIN_ATTEMPT_INFO",
+                "ST_LOGS THIRDPARTYPASSWORDLESS OVERRIDE GET_LOGIN_ATTEMPT_INFO",
```

Comment:
there was no way to get code match the expected outputs by tests, because web-js also calls getLoginAttemptInfo and we can not get rid of them, so calls are duplicated

Reply:
Please see why it’s happening twice. Is one coming from auth-react and one from web-js or both from web-js? Can you clearly explain please